config/api.yml

```diff
@@ -831,6 +834,21 @@
       :get:
       - :name: read
         :identifier: miq_report_view
+  :schedules:
+    :description: Report schedules
+    :identifier: miq_report_reports
+    :options:
+    - :subcollection
+    :methods: *70174834086080
```

Comment:
Since https://github.com/ManageIQ/manageiq/pull/9420 has been merged, you'll need to rename this `:verbs:`